deploy/auth-delegator.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
This has moved to v1: https://github.com/kubernetes/kubernetes/pull/49642

Reply:
so fast:D